src/EditorFeatures/Core/Extensibility/NavigationBar/NavigationBarItem.cs

```diff
@@ -82,8 +64,18 @@ public bool Equals(NavigationBarItem? other)
                    Grayed == other.Grayed &&
                    Indent == other.Indent &&
                    ChildItems.SequenceEqual(other.ChildItems) &&
-                   Spans.SequenceEqual(other.Spans) &&
-                   EqualityComparer<TextSpan?>.Default.Equals(NavigationSpan, other.NavigationSpan);
+                   Spans.SequenceEqual(other.Spans);
+        }
+    }
+
+    internal static class NavigationBarItemExtensions
+    {
+        public static TextSpan GetCurrentNavigationSpan(this NavigationBarItem item, ITextVersion toVersion, TextSpan span)
```

Comment:
nit:

```suggestion
        public static TextSpan GetCurrentNavigationItemSpan(this NavigationBarItem item, ITextVersion toVersion, TextSpan span)
```

The current naming reads misleading to me, but that may be holdover from seeing `NavigationSpan` on `NavigationBarItem` previously

Reply:
wfm.